templates/web/base/report/_update_state.html

```diff
@@ -1,5 +1,5 @@
 [% # Small chance of duplicates in the case of fixed - user followed by fixed - council %]
-[% SET update_state = update.problem_state %]
+[% SET update_state = update.problem_state_processed %]
 [% IF ( update_state AND update_state != global.last_state AND NOT (global.last_state == "" AND update.problem_state == 'confirmed') )
```

Comment:
I guess for consistently end of this line should use problem_state_processed as well?